scripts/_common.sh

```diff
@@ -4,8 +4,10 @@
 # COMMON VARIABLES
 #=================================================
 
+phpversion="7.4"
```

Comment:
You must use `YNH_PHP_VERSION` instead of phpversion

https://github.com/YunoHost/yunohost/blob/6fc6a2ba4c17434803cea2553a3dc7c51cedd338/data/helpers.d/php#L3-L6

`ynh_add_fpm_config` use it to define the php version to use

Reply:
Fixed